### DISCUSSION TITLE
Infinite scroll

### DISCUSSION THREAD

**Discussion by zt64** [community/community#66180] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The	current feed shows a MORE button at the very bottom, which can get annoying having to start and stop to click the button. But it would be better to instead just load once the user scrolls that far down. Maybe as an option to toggle between the two behaviors.

**Comment by jeremy-coleman** [comment]
We need a Cortana integration for voice control
